Quick_Start.ipynb

```diff
@@ -462,9 +531,9 @@
    "id": "1d783e02",
    "metadata": {},
    "source": [
-    "If you want to access the data for all the iterations, you can use `get_all()` or if you only wanted the converged iteration, you can use `get_converged()`.\n",
+    "If you want to access the data for all the iterations, you can use `get_all()` or if you only wanted the converged iteration, you can use `get_converged()`. In this specific case, both calls would deliver the same output, since in the benzene `inp` file the `OutputDuringSCF` parameter is not set to `True`, so that we only can access to the last converged iteration. For these last two methods, the user doesn't need to provide a `steps` method.\n",
```

Comment:
Shouldnt we specify the source in the previous cell?

Im getting the error 

> ValueError: There is more than one available source for the selected file.

When I run the previous cell.

```
density_data = run.default.scf.density.get(steps=1)
density_data
```